docs/script-api.md

```diff
@@ -275,6 +277,9 @@ Short for [spawn_layer_door](#spawn_layer_door).
 ### [`spawn_apep`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=spawn_apep)
 `int spawn_apep(float x, float y, LAYER layer, bool right)`<br/>
 Spawns apep with the choice if it going left or right, if you want the game to choose use regular spawn functions with `ENT_TYPE.MONS_APEP_HEAD`
+### [`spawn_tree`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=spawn_tree)
```

Comment:
Because someone asked for this specifically and I quickly found a function in the game that does it. Dregu and me discussed on Discord to make a general function that does this kinda stuff based on the entity type, but right now we don't have much more.